managers/bosh-manager/DirectorService.js

```diff
@@ -73,6 +78,25 @@ class DirectorService extends BaseDirectorService {
       });
   }
 
+  verifyFeatureSupport(feature) {
+    if (!_.includes(this.agent.features, feature)) {
```

Comment:
Can we do it like `plan.agant.feature` ?

Then we can get it out of DirectorService and make it a common util function?

Reply:
done